utils/copyImagesToQuay.txt

```diff
@@ -0,0 +1,20 @@
+# list images to copy to quay.io/eclipse/che-*, 1 per line
+# lines starting with # will be ignored 
+# see copyImagesToQuay.sh for skopeo copy script
+
+docker.io/centos/mongodb-36-centos7
+docker.io/centos/mysql-57-centos7
+docker.io/centos/postgresql-96-centos7
+docker.io/dirigiblelabs/dirigible-openshift:3.4.0
+docker.io/ksmster/che-editor-jupyter:5.7.0
+docker.io/traefik:v2.2.8
+docker.io/wsskeleton/eclipse-broadway
```

Comment:
we can move it from the eclipse Broadway repo

Reply:
great, let's do that. Meanwhile...

https://quay.io/repository/eclipse/che--wsskeleton--eclipse-broadway

Reply:
How could you copy without even approval of this PR ?!!!

Reply:
easily. I ran the script locally so that we could get the che release unblocked.

Reply:
this image should be removed from the script